lib/active_admin/views/components/active_admin_form.rb

```diff
@@ -72,10 +72,24 @@ def actions(*args, &block)
       end
 
       def commit_action_with_cancel_link
+        create_another_checkbox
         action(:submit)
         cancel_link
       end
 
+      def create_another_checkbox
+        if form_builder.object.new_record?
+          create_another = params[:create_another]
+          checkbox = Arbre::Context.new do
+            li do
+              input(create_another, checked: create_another, type: 'checkbox', id: 'create_another', name: 'create_another')
+              label(I18n.t('active_admin.create_another'), for: 'create_another')
+            end
+          end
```

Comment:
This hack is required since `ActiveAdminForm` delegates `input` method to form builder. Thus, I have to create new arbre context to bypass this override. Would you like to see this comment in the code?

Is there a simpler way to render checkbox here?